README.md

```diff
@@ -1,73 +1,130 @@
-    </a>
-</td>
-</tr>
-<tr>
-  <td>Build Status</td>
-  <td>
-    <a href="https://github.com/pathivu/pathivu/actions">
-    <img alt="Build Status" src="https://github.com/pathivu/pathivu/workflows/Rust/badge.svg" />
-    </a>
-  </td>
-</tr>
-<tr>
-	<td>Discord</td>
-	<td>
-		<a href="https://discord.gg/PGjRet">
-		<img src="https://img.shields.io/discord/628383521450360842.svg?logo=discord" />
-		</a>
-	</td>
-</tr>
-</table>
-
---- 
-
-Pathivu is powerful log ingestion and aggregation system. It's built from scratch by having cost-efficient and high write throughput in mind without trading log indexing. 
+<div align="center">
+	<img align="center" width="30%" src="https://github.com/pathivu/pathivu/blob/master/docs/img/pathivu.jpeg?raw=true">
+	<h1>
+		Pathivu: Logs you can search
+	</h1>
+</div>
 
-## Highlights
-- Fast Ingestion
-- Beautiful dashboard*
-- Log tailing
-- Log indexing
-- Cost-Efficient
-- Intuitive query language
 
-**  Not yet released. It's on the roadmap
+ Pathivu is a powerful and lightweight log ingestion and aggregation system.  It offers cost-efficiency and high throughput without trading away log indexing. It is perfect for cloud native workloads.
+
+
+
+<p align="center"><img width="90%" src="https://github.com/pathivu/pathivu/blob/master/docs/tail.gif?raw=true"/></p>
+
+<div align="center">
+<a href="https://github.com/pathivu/pathivu/blob/master/LICENSE">
+	 <img alt="License" src="https://img.shields.io/badge/License-Apache%202.0-blue.svg?style=flat-square&logo=appveyor">
+</a> <a href="https://discord.gg/PGjRet">
+	  <img src="https://img.shields.io/discord/628383521450360842.svg?logo=discord" />
+</a> <a href="https://docs.pathivu.io/#/">
+	   <img alt="View Documentation" src="https://img.shields.io/badge/docs-view%20documentation-orange?style=flat-square&logo=appveyor" />
+</a>
+<a href="https://github.com/pathivu/pathivu/actions">
+	   <img alt="Build Status" src="https://github.com/pathivu/pathivu/workflows/Rust/badge.svg" />
+</a>
+<a href="https://pathivu.io/#/">
+	   <img alt="View Website" src="https://img.shields.io/badge/website-view%20website-yellowgreen?style=flat-square&logo=appveyor" />
+</a>
+</div>
 
+## Index
+- [Highlights](#highlights)
+- [Chola](#chola)
+- [Usage](#usage)
+- [Katchi](#katchi)
+- [Documentation](https://docs.pathivu.io/#/)
+- [Website](https://pathivu.io)
+
+## Highlights
+- [X] Fast Ingestion
+- [ ] Beautiful dashboard*
+- [X] Log tailing
+- [X] Log indexing
+- [X] Cost-Efficient
+- [X] Intuitive query language
+- [X] Multi-threaded log ingestion
+- [X] Structured logging replayer
+- [X] Log retention 
 
 
-<p align="center"><img src="docs/tail.gif?raw=true"/></p>
+## Chola
+Chola is the service that Pathivu runs. It offers a gRPC service for fast log ingestion and an HTTP(s) backend for log querying and aggregation. By default, log ingestion runs on gRPC port `6180` and querying on HTTP(s) port `5180`. 
```

Comment:
Explain the component of pathivu stack.
Pathivu server - high performant log ingestion server where you can ingest logs for tbs of scale and query logs in efficient way. (both http and grpc have query and ingestion endpoints)
katchi - CLI tool to interact with pathivu server. add doc how to download and move it to /usr/bin....
and also we have `pathivu_HOST` enviromental variable. If it is specified no need for host option
I'll change `pathivu_HOST` to `PATHIVU_HOST` change according to that
pathivu-web - UI dashboard ....